### PR TITLE
[DEV-659] - Add app io overview

### PR DESCRIPTION
**Pull request by MarcoPonchia**
#### List of Changes

Add AppIO overview page:
- hero component

#### Motivation and Context

User can navigate to AppIO overview page

#### How Has This Been Tested?

User can navigate to http://localhost:3000/app-io/overview and see AppIO overview page

#### Screenshots (if appropriate):
screenshot-localhost_3000-2023 06 30-17_02_46

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/pages/[productSlug]/overview.tsx [hunk 1]
```diff
@@ -17,6 +17,8 @@ export type OverviewPageProps = {
   readonly path: string;
   readonly product: Product;
   readonly hero: {
+    readonly backgroundImage: string;
+    readonly altText?: string;
```
Comment (kin0992) [inline apps/nextjs-website/src/pages/[productSlug]/overview.tsx:21]:
```suggestion
    readonly altText: string;
```
a key factor for this developer portal is accessibility, so if the `backgroundImage` is not optional, I think nor the `altText` should be

**Review by kin0992 — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/pages/[productSlug]/overview.tsx (hunk 1 above)
Comment (datalek) [inline apps/nextjs-website/src/pages/[productSlug]/overview.tsx:21]:
```suggestion
    readonly altText: string;
```

File: apps/nextjs-website/src/_contents/ioSign/overview.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ export const ioSignOverview: OverviewData = {
   ...ioSignOverviewPath,
   product: ioSign,
   hero: {
+    backgroundImage: '/images/hero.jpg',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/ioSign/overview.ts:9]:
```suggestion
    altText: 'Hero: Raccogli i servizi del tuo ente in unico spazio.',
    backgroundImage: '/images/hero.jpg',
```

File: apps/nextjs-website/src/_contents/pagoPa/overview.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ export const pagoPaOverview: OverviewData = {
   ...pagoPaOverviewPath,
   product: pagoPA,
   hero: {
+    backgroundImage: '/images/hero.jpg',
```
Comment (datalek) [inline apps/nextjs-website/src/_contents/pagoPa/overview.ts:9]:
```suggestion
    altText: 'Hero: La piattaforma per gestire e rendicontare i pagamenti.',
    backgroundImage: '/images/hero.jpg',
```

**Review by datalek — CHANGES_REQUESTED** [review]

**Review by kin0992 — APPROVED** [review]

**Pull request merged**